### DISCUSSION TITLE
Introducing New Collaborator

### DISCUSSION THREAD

**Discussion by lllyasviel** [lllyasviel/stable-diffusion-webui-forge#1297] [Announcements]
Hey Forgers,

@Panchovix is now on Forge repo with writing permission.

Forge

**Comment by AcademiaSD** [comment]
Bravo!!!!

**Comment by vicento** [comment]
bienvenue

**Comment by Panchovix** [comment]
Many thanks! I will start to slowly add some new features and make sure they work as expected.

**Reply by ishadowx27** [reply]
This is the best course of action. Forge + reForge = BeastForge 🦾🙂

**Reply by queenofinvidia** [reply]
> Many thanks! I will start to slowly add some new features and make sure they work as expected.

Awesome!! Thank you! I look forward to see what you guys will come up with/fix in the future <3 more great minds working together is always a good thing!

**Reply by Paharven-Salin** [reply]
> Many thanks! I will start to slowly add some new features and make sure they work as expected.

Great news, this is even better, Bionic Forge Rebirth Apocalypse is GO! Respect and make it a LARGE coffee 🥇 

![yay-imessage-sticker-330968-3822546170](https://github.com/user-attachments/assets/cd3ebd27-246d-4f14-81ee-a338b6609b88)

**Comment by altoiddealer** [comment]
Amazing news!

**Comment by ZeroCool22** [comment]
![335707206_700](https://github.com/user-attachments/assets/c62496a5-492b-415c-ba11-decb3f306078)

**Comment by stripealipe** [comment]
Love this and thank you both for your hard work - it really IS appreciated!!

**Comment by yiyi147258** [comment]
Hello developers, I hope you can find time in your busy schedule to update the controlnet application of Flux.

**Reply by altoiddealer** [reply]
![Screenshot 2024-08-20 091233](https://github.com/user-attachments/assets/350d2240-57bc-4a86-8fb7-80cc847bbe42)

They know

**Comment by rlewisfr** [comment]
Many thanks to everyone working on this project.  I KNEW I didn't have to drink the Comfy Kool-Aid!

**Comment by yamfun** [comment]
wow and please Flux ControlNet

**Comment by minsky91** [comment]
Great news! Would somebody post it on the SD subreddit? Will make a lot of folks happy.
